ide/che-core-ide-api/src/main/java/org/eclipse/che/ide/api/editor/keymap/KeyBindingAction.java

```diff
@@ -16,5 +16,5 @@
 public interface KeyBindingAction {
 
     /** The triggered action. */
-    void action();
+    boolean action();
```

Comment:
That was the tricky part. Have a look at the call chain starting with `OrionEditorWidget.addKeyBinding(KeyBinding, String)` where the `OrionTextViewOverlay.setAction(String, Action, String)` is called, which will then create a `function` delegate in JS to return `Action.onAction()`'s boolean return value at [l168](https://github.com/TypeFox/che/blob/aa5a9368165db032c8568626c41712b11187dd86/plugins/plugin-orion/che-plugin-orion-editor/src/main/java/org/eclipse/che/ide/editor/orion/client/jso/OrionTextViewOverlay.java#L168). 

Passing the boolean result along this call chain was the key to override a bound `autoPairAngleBracket` action in order to _disable_ the auto insert of the closing bracket. Basically, if the action always returns true, the key event handler of `orion.editor.TextView` won't proceed with the default event handling, which makes sure the opening bracket is inserted at all. The overriding `KeyBindingAction` may return `false` to re-enable the default handling.